README.MD

```diff
@@ -14,8 +14,10 @@ This is the Go version of the [onboarding exercise](https://github.com/wexinc/ps
 - [Logrus](https://github.com/Sirupsen/logrus)
 
 ## Pre-requisites
-- Docker
-- Go
+- [Go](https://golang.org/doc/install)
+- [Docker](https://docs.docker.com/manuals/)
+- [Docker-compose](https://docs.docker.com/compose/)
+- [Swagger](https://github.com/swaggo/gin-swagger)
```

Comment:
Did you create a technical proposal to use swagger? ONe is needed since none of our applications use gin-swagger to allow you to test the API.

Reply:
No, I need to create one